### DISCUSSION TITLE
Richiesta chiarimenti su 'ulterioreCittadinanza' per il caso d'uso Citt_029.

### DISCUSSION THREAD

**Discussion by gcuci** [italia/ansc#1060] [Supporto servizi cooperativi]
Buonasera,

nel file di mapping del caso d'uso Citt_029 ('Trascrizione: Esito dell'accertamento per chi avendo perduto la cittadinanza intende riacquistarla mediante prestazione del servizio militare per lo stato italiano'), sono presenti i due campi obbligatori:

evento.trascrizioneCittadinanza.idUlterioreCittadinanza
evento.trascrizioneCittadinanza.ulterioreCittadinanza

che dovrebbero corrispondere all'id ed alla descrizione delle cittadinanze indicate.

Per questi, non troviamo una corrispondenza nei nomi presenti nello swagger del ModelTrascrizioneCittadinanza, dove ci sono:

- ulterioreCittadinanza (che secondo il valore di esempio dovrebbe corrispondere ad un id);
- descrizioneUlterioreCittadinanza (che dovrebbe essere la descrizione).

Inoltre, se si riferiscono alla cittadinanza ulteriore (quella attuale del cittadino), oltre a quella italiana del riacquisto, come mai non sono presenti anche negli altri casi d'uso simili? (es. Citt_31 'Trascrizione: Esito dell'accertamento per chi, avendo perduto la cittadinanza italiana, intende riacquistarla per l'assunzione di pubblico impiego alle dipendenze dello Stato italiano' o Citt_034 'Trascrizione: Riacquisto per residenza in Italia esito dell'accertamento', ...).

Potete darci un chiarimento?

In attesa di riscontro,

cordiali saluti.

**Comment by asqueglia** [comment]
Buongiorno, prendiamo in carico la segnalazione e trasformiamo in issue questa discussion chiudendola. La preghiamo di seguire l'evoluzione della issue #1385
